File: 프로그래머스/level1/문자열 다루기 기본.cpp
#include <string>
#include <vector>
using namespace std;

bool solution(string s) {
    int len = s.length();
    if(len != 4 && len != 6) return false;
    
    for(int i = 0;i < s.length();i++)
        if(s.at(i) - '0' > 10 || s.at(i) < 0) return false;
    return true;
}

File: programmers/level1/문자열 다루기 기본.java
import java.util.*;

class Solution {
    public boolean solution(String s) {
        int l = s.length();
        if(l != 4 && l != 6) return false;
        
        for(int i = 0;i < l;i++)
            if(!Character.isDigit(s.charAt(i))) return false;
    
        return true;
    }
}